command/issue.go

```diff
@@ -383,7 +383,7 @@ func issueCreate(cmd *cobra.Command, args []string) error {
 
 	action := SubmitAction
 
-	interactive := title == "" || body == ""
+	interactive := !cmd.Flags().Changed("title") && !cmd.Flags().Changed("body")
```

Comment:
Based on a discussion with my team, let's change this to 
```suggestion
	interactive := !(cmd.Flags().Changed("title") && cmd.Flags().Changed("body"))
```

This is to enforce that both `--title` and `--body` need to be passed explicitly (even if empty string) for the interactive mode to be disabled.